src/System.Net.NameResolution/tests/PalTests/NameResolutionPalTests.cs

```diff
@@ -35,6 +35,7 @@ public void TryGetAddrInfo_LocalHost()
             Assert.NotNull(hostEntry.Aliases);
         }
 
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotArm64Process))] // [ActiveIssue(32797)]
         [Fact]
```

Comment:
You'll need to remove the second [Fact] attribute here.
```suggestion
```

Reply:
Ugh, yes, thanks.